src/presentation/components/Template/Header/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Styles from './styles.scss'
+import Navigation from '../Navigation'
+
+const Header: React.FC = () => {
+  return (
+    <div className={Styles.Container}>
+      <img
+        className={Styles.logo}
+        src="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg"
```

Comment:
I suggest you to put something written like "My App", as I did here https://main--cheery-shortbread-5efb5b.netlify.app/

it's easier to change